tests/foreman/cli/test_hammer.py

```diff
@@ -183,6 +183,9 @@ def test_positive_check_debug_log_levels(target_sat):
     # Verify value of `log4j.logger.org.candlepin` as `DEBUG`
     result = target_sat.execute('grep DEBUG /etc/candlepin/candlepin.conf')
     assert result.status == 0
+    assert (
+        'log4j.logger.org.candlepin = DEBUG' or 'log4j.logger.org.candlepin=DEBUG' in result.stdout
+    )
```

Comment:
Instead you can run `grep log4j.logger.org.candlepin /etc/candlepin/candlepin.conf` and assert DEBUG in result.stdout?

Reply:
@Gauravtalreja1 suggestion applied, thanks